gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-users/src/main/java/io/gravitee/am/gateway/handler/users/service/DomainUserConsentFacade.java

```diff
@@ -24,34 +24,31 @@
 import java.util.Set;
 
 /**
+ * Manages gateway operations on users within a single domain
+ *
  * @author Titouan COMPIEGNE (titouan.compiegne at graviteesource.com)
  * @author GraviteeSource Team
  */
-public interface UserService {
-
-    Maybe<User> findById(String id);
+public interface DomainUserConsentFacade {
```

Comment:
Not sure about the Facade sufix. 

```suggestion
public interface DomainUserConsentService {
```

Reply:
I mean we can call it something else if you have a better idea :)
"Facade" is just the term I've seen used for this kind of service most often. "Adapters" is another one. The suffix doesn't matter to me that much as long as it's not "service".

My aim is to evolve the code towards a situation where we have clear boundaries between:
*  services that are "internal" to the application
* HTTP code that handles request parsing and response generation (i.e. handlers, resources)
* and "services" that implement logic the entrypoints need using the internal services (facades/adapters)

This will give us better separation of concerns, which should help with development velocity and just make making sense of the code easier.

Reply:
Ok, may I ask you to discuss your aims with all the team to define a set of rules and be align to the target ?
I'm agree with the fact that there is matter of improvement but we need to discuss it first.

Reply:
I'll revert to the `Service` suffix for now, this should be discussed independent of this PR